web/client/components/mapcontrols/searchbookmarkconfig/AddNewBookmark.jsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
```

Comment:
hi, missing copyrights for new file, please add to all new file that are missing it, thanks